src/features/organizations/l10n/messageIds.ts

```diff
@@ -1,6 +1,20 @@
 import { m, makeMessages } from 'core/i18n';
 
 export default makeMessages('feat.organizations', {
+  gen3: {
+    description: m(
+      'This is the new (generation 3) organizer web app. If you are used to the old one, you will find lots of new features and an improved user interface here. But if you want, you can still use the old organizer app for a while longer.'
+    ),
+    gen2Button: m('Go to the old interface'),
+    title: m('Welcome to the new Zetkin!'),
+  },
+  notOrganizer: {
+    description: m(
+      'This part of Zetkin is only available to users who have been granted organizer access my another organization official. You do not currently have the sufficient level of access. If you believe that this is a mistake, please reach out to your local Zetkin officials or to Zetkin Foundation.'
```

Comment:
Very tiny typo: should be "by" instead of "my"

Reply:
Thank you! Update coming in a minute. 💯

Reply:
Fixed in 7b5bbf6044633fbb1c40833c679e4f038d1e8c43. Please review again. 🙏